runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java

```diff
@@ -182,6 +184,48 @@ void rebuild()
 		repaint();
 	}
 
+	void collapse()
+	{
+		if (!isCollapsed())
+		{
+			itemContainer.setVisible(false);
+			applyDimmer(false, logTitle);
+		}
+	}
+
+	void expand()
+	{
+		if (isCollapsed())
+		{
+			itemContainer.setVisible(true);
+			applyDimmer(true, logTitle);
+		}
+	}
+
+	boolean isCollapsed()
+	{
+		return itemContainer.isVisible() == false;
+	}
+
+	private void applyDimmer(boolean brighten, JPanel panel)
+	{
+		for (Component component : panel.getComponents())
+		{
+			Color color = component.getForeground();
+
+			if (brighten)
```

Comment:
I'd simplify this with a ternary

..setForeground(brighten ? color.brighter() : color.darker());